.eslintrc

```diff
@@ -1,102 +1,43 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb",
+  "extends": [
+    "airbnb",
```

Comment:
btw unless you're using react, you should use `eslint-config-airbnb-base` which will avoid pulling in `eslint-plugin-react` and `eslint-plugin-jsx-a11y`